src/testUtils.ts

```diff
@@ -166,7 +166,7 @@ export function goTest(testconfig: TestConfig): Thenable<boolean> {
 			outputChannel.show(true);
 		}
 
-		let buildTags: string = testconfig.goConfig['buildTags'];
+		let buildTags: string = testconfig.goConfig['testTags'];
```

Comment:
I believe we should fallback to using `buildTags` if no `testTags` are provided to ensure we are not breaking anybody else's use case.

Reply:
Of course!